packages/abi-coder/src/encoding/coders/AbstractCoder.ts

```diff
@@ -26,9 +26,8 @@ export type DecodedValue =
   | { [key: string]: DecodedValue }
   | Record<string, Primitive>;
 
-export type TypesOfCoder<TCoder> = TCoder extends Coder<infer TInput, infer TDecoded>
-  ? { Input: TInput; Decoded: TDecoded }
-  : never;
+export type TypesOfCoder<TCoder> =
```

Comment:
Since we had to do some linting updates here should the changesets bump these as `patches` ?

Reply:
Good catch. https://github.com/FuelLabs/fuels-ts/pull/2584/commits/45e9840b3fae4669e254e771843bbe901ff08fd9